ntoskrnl/mm/ARM3/mminit.c

```diff
@@ -1809,12 +1809,12 @@ MiBuildPagedPool(VOID)
     //
     // Now check how many PTEs will be required for these many pages.
     //
-    Size = (Size + (1024 - 1)) / 1024;
+    Size = (Size + (PTE_COUNT - 1)) / PTE_COUNT;
```

Comment:
Can we standardize on PTE_PER_PAGE instead? Its name seems clearer and PTE_COUNT isn't used anywhere right now.